pattoo/api/portal/react/src/Logout.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Logout extends React.Component {
+    constructor(props){
+        super(props);
+    }
+
+    componentDidMount(){
+        let self = this;
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.